_posts/2019-02-01-dynamic-spawn-scheduler.md

```diff
@@ -1,12 +1,12 @@
 ---
 layout: posts
-title: Dynamic scheduling for faster remote builds
+title: Dynamic scheduling for faster builds
 authors:
   - jmmv
   - jin
 ---
 
-The recently-released [Bazel 0.21](https://blog.bazel.build/2018/12/19/bazel-0.21.html) delivers a new feature that can make your remote builds faster by making use of local resources where possible. Prior to this release, this was already doable to some extent by using manual strategy definitions, but this new feature takes all manual decisions out of the picture.
+The recently-released [Bazel 0.21](https://blog.bazel.build/2018/12/19/bazel-0.21.html) delivers a new feature that can make your builds faster by making use of remote *and* local resources transparently. Prior to this release, this was already doable to some extent by using manual strategy definitions, but this new feature takes all manual decisions out of the picture.
```

Comment:
I suggest to tone down this paragraph even more to mention that is an experiment, not a "delivered new feature"

Reply:
Done.